### PR TITLE
CVE Record Aug 1st

### PR DESCRIPTION
**Pull request by monicamurray**

### CODE REVIEW

File: 2021/27xxx/CVE-2021-27551.json [hunk 1]
```diff
@@ -1,18 +1,85 @@
 {
```
Comment (cve-team) [inline 2021/27xxx/CVE-2021-27551.json:1]: CNA(s) of user (HCL) do not match owning CNA.

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

File: 2021/27xxx/CVE-2021-27551.json (hunk 1 above)
Comment (monicamurray) [reply 2021/27xxx/CVE-2021-27551.json:1]: Can you point to the mismatch?  Which fields?

Comment (monicamurray) [reply 2021/27xxx/CVE-2021-27551.json:1]: Sorry.  I do not understand the problem, so not sure how to resolve it.

Comment (monicamurray) [reply 2021/27xxx/CVE-2021-27551.json:1]: I see the error.  The CVE ID should be CVE-2022-27551.  Let me correct that

**Comment by monicamurray** [comment]
I have reached out several times both here and via email, but have not gotten any further direction. I will just try again with a new branch.

**Comment by monicamurray** [comment]
Hello CVE Team,

I realized that I made changes to the wrong CVE ID which does not belong to HCL Software.
Modified:  CVE-2021-27551
Should have been: CVE-2022-27551

How do I restore the other document which was modified erroneously?
I corrected the mistake and pushed the changes (on the same branch).

Thanks in advance!

Regards,


[9k=]
Monica (Senior) Murray
PSIRT Program Manager &
Senior Software Engineer – Product Security
HCL Software<http://www.hcltechsw.com/>



From: CVE Team ***@***.***>
Reply-To: CVEProject/cvelist ***@***.***>
Date: Monday, August 1, 2022 at 11:05 AM
To: CVEProject/cvelist ***@***.***>
Cc: Monica A Murray ***@***.***>, Author ***@***.***>
Subject: Re: [CVEProject/cvelist] CVE Record Aug 1st (PR #6719)

[CAUTION: This Email is from outside the Organization. Unless you trust the sender, Don’t click links or open attachments as it may be a Phishing email, which can steal your Information and compromise your Computer.]

@cve-team requested changes on this pull request.

Auto validation errors:

________________________________

In 2021/27xxx/CVE-2021-27551.json<https://apc01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FCVEProject%2Fcvelist%2Fpull%2F6719%23discussion_r934635220&data=05%7C01%7Cmonica.murray%40hcl.com%7Cef8222b4e0f24295970a08da73cf4227%7C189de737c93a4f5a8b686f4ca9941912%7C0%7C0%7C637949631246493356%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=gxY9pfgvVEL7SjC%2FMl84KoREHijjr%2FbHylkSLDXyHl8%3D&reserved=0>:

> @@ -1,18 +1,85 @@

 {

CNA(s) of user (HCL) do not match owning CNA.

—
Reply to this email directly, view it on GitHub<https://apc01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FCVEProject%2Fcvelist%2Fpull%2F6719%23pullrequestreview-1057456219&data=05%7C01%7Cmonica.murray%40hcl.com%7Cef8222b4e0f24295970a08da73cf4227%7C189de737c93a4f5a8b686f4ca9941912%7C0%7C0%7C637949631246493356%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=N%2B%2BGMhO1euD7p67qzuze5sfAYENy%2F0NgzIa4bM6ZnjU%3D&reserved=0>, or unsubscribe<https://apc01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FARUX7XLAZIN5IHQXEWHI4N3VW7RTBANCNFSM55H4FTNA&data=05%7C01%7Cmonica.murray%40hcl.com%7Cef8222b4e0f24295970a08da73cf4227%7C189de737c93a4f5a8b686f4ca9941912%7C0%7C0%7C637949631246493356%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=oR48gWQENm3jf2JZ%2BM6BAivEKCMpJDEHq8C%2FjceXIuc%3D&reserved=0>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

::DISCLAIMER::
________________________________
The contents of this e-mail and any attachment(s) are confidential and intended for the named recipient(s) only. E-mail transmission is not guaranteed to be secure or error-free as information could be intercepted, corrupted, lost, destroyed, arrive late or incomplete, or may contain viruses in transmission. The e mail and its contents (with or without referred errors) shall therefore not attach any liability on the originator or HCL or its affiliates. Views or opinions, if any, presented in this email are solely those of the author and may not necessarily reflect the views or opinions of HCL or its affiliates. Any form of reproduction, dissemination, copying, disclosure, modification, distribution and / or publication of this message without the prior written consent of authorized representative of HCL is strictly prohibited. If you have received this email in error please delete it and notify the sender immediately. Before opening any email and/or attachments, please check them for viruses and other defects.
________________________________

**Comment by monicamurray** [comment]
Hello CVE Team,

I updated the wrong CVE, but have corrected that mistake and submitted the CVE record again.

It has not been merged.  May I request help with resolving this issue.


Regards,


[9k=]
Monica (Senior) Murray
PSIRT Program Manager &
Senior Software Engineer – Product Security
HCL Software<http://www.hcltechsw.com/>



From: CVE Team ***@***.***>
Reply-To: CVEProject/cvelist ***@***.***>
Date: Monday, August 1, 2022 at 11:05 AM
To: CVEProject/cvelist ***@***.***>
Cc: Monica A Murray ***@***.***>, Author ***@***.***>
Subject: Re: [CVEProject/cvelist] CVE Record Aug 1st (PR #6719)

[CAUTION: This Email is from outside the Organization. Unless you trust the sender, Don’t click links or open attachments as it may be a Phishing email, which can steal your Information and compromise your Computer.]

@cve-team requested changes on this pull request.

Auto validation errors:

________________________________

In 2021/27xxx/CVE-2021-27551.json<https://apc01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FCVEProject%2Fcvelist%2Fpull%2F6719%23discussion_r934635220&data=05%7C01%7Cmonica.murray%40hcl.com%7Cef8222b4e0f24295970a08da73cf4227%7C189de737c93a4f5a8b686f4ca9941912%7C0%7C0%7C637949631246493356%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=gxY9pfgvVEL7SjC%2FMl84KoREHijjr%2FbHylkSLDXyHl8%3D&reserved=0>:

> @@ -1,18 +1,85 @@

 {

CNA(s) of user (HCL) do not match owning CNA.

—
Reply to this email directly, view it on GitHub<https://apc01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FCVEProject%2Fcvelist%2Fpull%2F6719%23pullrequestreview-1057456219&data=05%7C01%7Cmonica.murray%40hcl.com%7Cef8222b4e0f24295970a08da73cf4227%7C189de737c93a4f5a8b686f4ca9941912%7C0%7C0%7C637949631246493356%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=N%2B%2BGMhO1euD7p67qzuze5sfAYENy%2F0NgzIa4bM6ZnjU%3D&reserved=0>, or unsubscribe<https://apc01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FARUX7XLAZIN5IHQXEWHI4N3VW7RTBANCNFSM55H4FTNA&data=05%7C01%7Cmonica.murray%40hcl.com%7Cef8222b4e0f24295970a08da73cf4227%7C189de737c93a4f5a8b686f4ca9941912%7C0%7C0%7C637949631246493356%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=oR48gWQENm3jf2JZ%2BM6BAivEKCMpJDEHq8C%2FjceXIuc%3D&reserved=0>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

::DISCLAIMER::
________________________________
The contents of this e-mail and any attachment(s) are confidential and intended for the named recipient(s) only. E-mail transmission is not guaranteed to be secure or error-free as information could be intercepted, corrupted, lost, destroyed, arrive late or incomplete, or may contain viruses in transmission. The e mail and its contents (with or without referred errors) shall therefore not attach any liability on the originator or HCL or its affiliates. Views or opinions, if any, presented in this email are solely those of the author and may not necessarily reflect the views or opinions of HCL or its affiliates. Any form of reproduction, dissemination, copying, disclosure, modification, distribution and / or publication of this message without the prior written consent of authorized representative of HCL is strictly prohibited. If you have received this email in error please delete it and notify the sender immediately. Before opening any email and/or attachments, please check them for viruses and other defects.
________________________________

**Pull request closed without merging**